automation/components-functests.setup.sh

```diff
@@ -99,6 +99,15 @@ git-utils::fetch_component ${component_path} ${component_url} ${component_commit
 
 export TMP_COMPONENT_PATH=${component_path}
 
+# TODO remove
```

Comment:
Either remove this or provide enough context to make this actionable

Reply:
https://github.com/kubevirt/cluster-network-addons-operator/pull/1807#discussion_r1664287961

Reply:
removed, lets see all pass (required PRs were merged on OVS/KMP)